controllers/status/status.go

```diff
@@ -210,7 +216,135 @@ func RemoveComponentCondition(conditions *[]conditionsv1.Condition, component st
 	conditionsv1.RemoveStatusCondition(conditions, conditionsv1.ConditionType(component+ReadySuffix))
 }
 
-// ModelRegistryStatus struct holds the status for the ModelRegistry component.
+// +k8s:deepcopy-gen=true
+type ComponentReleaseStatus struct {
+	DisplayName string                  `json:"displayname,omitempty"`
+	Version     version.OperatorVersion `json:"version,omitempty"`
+	RepoURL     string                  `json:"repourl,omitempty"`
```

Comment:
The JSON keys here could maybe be slightly improved (it's probably just me, but I had a tendancy to read `repourl` as "repour L" rather than "repo URL" :sweat_smile:)

```suggestion
	DisplayName string                  `json:"displayName,omitempty"`
	Version     version.OperatorVersion `json:"version,omitempty"`
	RepoURL     string                  `json:"repoURL,omitempty"`
```

Reply:
Related to this, your screenshot in the description also has a "name" field, where does that come from, or is it no longer included? If it's no longer included, then maybe displayName here should just become name?

Reply:
here the field "name" would contain component name, which is already present a level above(like below) so we wouldnt need have "name" field again. I will update the screenshot now.

```
status:
   components:
        codeflare:
             - displayname: 
                version:
                repoURL:
```

Reply:
That's the name of the component from the ODH operator POV, but the name/displayname for each element in the list is the name/displayname for the "upstream release project" (or whatever), right? So I'd suggest changing the "displayname" field of the elements in the "releases" list here to just be "name" if there's just going to be one key to represent this.

Reply:
yea makes sense. will change it to "name"